subjects/displays/README.md

```diff
@@ -0,0 +1,19 @@
+## displays
+
+### Instructions
+
+Write a program that displays an `S` character on the standard followed by newlines (`'\n'`)
```

Comment:
Instructions need to be fixed. I would suggest --> "Write a program that displays a character `S` followed by a newline `\n`."